.gitignore

```
@@ -13,3 +13,5 @@ notebooks/*
 examples/.ipynb_checkpoints/*
 .pytest_cache/*
 pytest/.pytest_cache/
+doc/tutorials_gbs/*
+examples_gbs/*.html
```

```suggestion
examples_gbs/*.html
doc/_static/thumbs/
```

Thanks @josh146! Also wanted to ask - I see that sphinx-gallery makes some thumbnails in `doc/_static/thumbs/` - generally we want to add these to the repo or not? I guess we can skip as it's automatically made during build. So how about we add `doc/_static/thumbs/` to the `.gitignore`? And also remove the `doc/_static/thumbs/code.png` currently in this PR.

---

I notice PL has this:
https://github.com/XanaduAI/pennylane/tree/master/doc/_static

---

Good point. We _do_ want to have `_static/thumbs/code.png` in the repository, as this is the default thumbnail if none is provided (we can change the actual image later). But anything else that is created and added to this directory we want to ignore.

We can add this to the gitignore:
```
doc/_static/thumbs/*
!doc/_static/code.png
```

---

```suggestion
examples_gbs/*.html
doc/_static/thumbs/*
!doc/_static/code.png
```
Nice!